nuxeo-java-client-test/src/test/resources/test-portal-sso-login-contrib.xml

```diff
@@ -0,0 +1,62 @@
+<component name="org.nuxeo.ecm.platform.authenticator.portal.sso.config">
+
+    <require>org.nuxeo.ecm.platform.ui.web.auth.WebEngineConfig</require>
+    <require>org.nuxeo.ecm.platform.login.Portal</require>
+    <require>org.nuxeo.ecm.automation.server.auth.config</require>
+
+    <require>org.nuxeo.ecm.restapi.server.auth.config</require> <!-- specificChains -->
+
+    <extension target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService" point="authenticators">
+        <authenticationPlugin name="PORTAL_AUTH">
+            <loginModulePlugin>Trusting_LM</loginModulePlugin>
+            <parameters>
+                <!-- define here shared secret between the portal and Nuxeo server -->
+                <parameter name="secret">nuxeo5secretkey</parameter>
+                <parameter name="maxAge">3600</parameter>
+            </parameters>
+        </authenticationPlugin>
+    </extension>
+
+    <!-- Include Portal Auth into authentication chain -->
+    <extension target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService" point="chain">
+        <authenticationChain>
+            <plugins>
+                <!-- Keep basic Auth at top of Auth chain to support RSS access via BasicAuth -->
+                <plugin>BASIC_AUTH</plugin>
+                <plugin>PORTAL_AUTH</plugin>
+                <plugin>FORM_AUTH</plugin>
+                <plugin>WEBENGINE_FORM_AUTH</plugin>
+                <plugin>ANONYMOUS_AUTH</plugin>
+                <plugin>WEBSERVICES_AUTH</plugin>
+            </plugins>
+        </authenticationChain>
+    </extension>
+
+    <extension point="specificChains" target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService">
```

Comment:
For the same reason as above you don't need to override the RestAPI specific authentication chain